network-peering/mongodb-atlas-networkpeering.json

```diff
@@ -0,0 +1,109 @@
+{
+    "typeName": "MongoDB::Atlas::NetworkPeering",
+    "description": "This resource allows to create, read, update and delete a network peering",
+    "sourceUrl": "https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/network-peering",
+    "definitions": {
+        "apiKeyDefinition": {
+            "type": "object",
+            "properties": {
+                "PublicKey": {
+                    "type": "string"
+                },
+                "PrivateKey": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        }
+    },
+    "properties": {
+        "ProjectId": {
+            "description": "The unique identifier of the project.",
+            "type": "string"
+        },
+        "ContainerId": {
+            "description": "Unique identifier of the Atlas VPC container for the AWS region.",
+            "type": "string"
+        },
+        "AccepterRegionName": {
+            "description": "AWS region where the peer VPC resides. Returns null if the region is the same region in which the Atlas VPC resides.",
+            "type": "string"
+        },
+        "AwsAccountId": {
+            "description": "AWS account ID of the owner of the peer VPC.",
+            "type": "string"
+        },
+        "PeerId": {
+            "description": "Unique Identifier of the Peer",
+            "type": "string"
+        },
+        "ProviderName": {
+            "description": "The name of the provider",
+            "type": "string"
+        },
+        "RouteTableCidrBlock": {
+            "description": "Peer VPC CIDR block or subnet.",
+            "type": "string"
+        },
+        "VpcId": {
+            "description": "Unique identifier of the peer VPC.",
+            "type": "string"
+        },
+        "ConnectionId": {
+            "description": "Unique identifier for the peering connection.",
+            "type": "string"
+        },
+        "ErrorStateName": {
+            "description": "Error state, if any.",
+            "type": "string"
+        },
+        "StatusName": {
+            "description": "The VPC peering connection status",
+            "type": "string"
+        },
+        "Id": {
+            "description": "The CIDR block that Atlas uses for the clusters in your project.",
```

Comment:
Looks incorrect

Reply:
I based from mongodb doc but I changed that.